### PR TITLE
Updated Footer Hyperlink

### LINKED ISSUE #3052: [Footer] Update Programs hyperlink text
**Current Behavior**
The "Programs" link in the site footer is nondescript.

Screen Shot 2022-08-09 at 3 50 10 PM

**Desired Situation**
"Internships" is a more accurate word for this hyperlink.

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by bensonOSei**
**Description**

This PR changes the **Programs** hyperlink in the footer section to **Internships**

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.
 
fixes #3052

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
Why?

**Comment by bensonOSei** [comment]
> Why?

It is an open issue from @leecalcote. You can check it out [here](https://github.com/layer5io/layer5/issues/3052)

**Comment by Nikhil-Ladha** [comment]
Hmm... I wonder if "Internship Programs" make more sense or not? 🤔
@leecalcote ?

**Comment by Nikhil-Ladha** [comment]
Also, while we are there please fix the DCO check following the steps mentioned [here](https://github.com/layer5io/layer5/pull/3053/checks?check_run_id=7759986753)

**Comment by David-Ottache** [comment]
Hi...I'm kinda new to this, basically learning open source and how to contribute.
Can I join in?

**Comment by leecalcote** [comment]
> Hmm... I wonder if "Internship Programs" make more sense or not? 🤔
> @leecalcote ?

It does. You’re right. It’s just… too darn long.

**Comment by Nikhil-Ladha** [comment]
> It does. You’re right. It’s just… too darn long.

Well...we have longer words in the footer as subitems 😬

**Comment by leecalcote** [comment]
> > It does. You’re right. It’s just… too darn long.
> 
> Well...we have longer words in the footer as subitems 😬

Oh, good point. :) Yes, if it fits, that'd be good. 👍

**Comment by Nikhil-Ladha** [comment]
@bensonOSei please update the text to `Internship Programs`.
It fit's perfectly, I have cross checked.

**Comment by l5io** [comment]
🚀 Preview for commit 5982c58a19e8e9cdc88a7a12cdac9a9c8ba9beab at: https://62f3e36ce63d5810120e857a--layer5.netlify.app

**Comment by bensonOSei** [comment]
> @bensonOSei please update the text to `Internship Programs`.
> It fit's perfectly, I have cross checked.

Alright... I think `Internship Programs` fits better than `Internships`. 
Working on it now

File: src/sections/General/Footer/index.js [hunk 1]
```diff
@@ -121,7 +121,7 @@ const Footer = () => {
                   </li>
                   <li>
                     <Link className="category-link" to="/careers/programs">
-                      Programs
+                      Internships Programs
```
Comment (Nikhil-Ladha) [inline src/sections/General/Footer/index.js:124]:
```suggestion
                      Internship Programs
```

**Comment by Nikhil-Ladha** [comment]
Just as I would about to click the button, huh!?

**Comment by leecalcote** [comment]
I had the same feeling of let's "git 'er done", I suspect. 🙂

**Pull request merged**